### PR TITLE
Fixed typos

### PR DESCRIPTION
**Pull request by NinjaCoderDev**
Fixed some typos in Readme.md and code_of_conduct.md

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 ## Our Pledge
 
 In the interest of fostering an open and welcoming environment, we as
-contributors and maintainers pledge to making participation in our project and
```
Comment (tedcasey) [inline CODE_OF_CONDUCT.md]: This file is all legalese.  Didn't change anything here.  Please leave as is.

File: README.md [hunk 2]
```diff
@@ -20,16 +20,16 @@
 
 # Overview
 
-Adobe Ride is a REST API Java test automation framework.  It is service-agnostic, modular and extensible, and built on top of industry-standard tools such as [Rest-Assured](http://rest-assured.io/), [TestNG](https://testng.org/doc/index.html), [Everit](https://github.com/everit-org/json-schema), and [Gatling](https://gatling.io/).  The goal of the framework is to speed development of tests and validation of APIs by abstracting repetitive code and standardizing calls for easier functional and end-to-end testing across micro-services 
+Adobe Ride is a REST API Java test automation framework.  It is service-agnostic, modular and extensible, and built on top of industry-standard tools such as [Rest-Assured](http://rest-assured.io/), [TestNG](https://testng.org/doc/index.html), [Everit](https://github.com/everit-org/json-schema), and [Gatling](https://gatling.io/).  The goal of the framework is to speed the development of tests and validation of APIs by abstracting repetitive code and standardizing calls for easier functional and end-to-end testing across micro-services
```
Comment (tedcasey) [inline README.md:23]: A bit stylistic, but ok.

File: README.md [hunk 3]
```diff
@@ -20,16 +20,16 @@
...
+Adobe Ride is a REST API Java test automation framework.  It is service-agnostic, modular and extensible, and built on top of industry-standard tools such as [Rest-Assured](http://rest-assured.io/), [TestNG](https://testng.org/doc/index.html), [Everit](https://github.com/everit-org/json-schema), and [Gatling](https://gatling.io/).  The goal of the framework is to speed the development of tests and validation of APIs by abstracting repetitive code and standardizing calls for easier functional and end-to-end testing across micro-services 
 
-Adobe Ride has already seen heavy use in a previous Adobe projects, and the intent is to utilize the experience there, leverage lessons learned, and allow the open-source community to in turn help develop it to make it more robust.
```
Comment (tedcasey) [inline README.md:25]: This has seen usage in more than one Adobe project so it is correct as is.

File: README.md [hunk 4]
```diff
@@ -20,16 +20,16 @@
...
+Adobe Ride has already seen heavy use in previous Adobe projects, and the intent is to utilize the experience there, leverage lessons learned, and allow the open-source community to in turn help develop it to make it more robust.
 
 You use Adobe ride by extending it in three primary ways:
 * Extending it's model-util to model rapidly generate and easily manipulate call payload data.
 * Provide some config information to point your tests at any endpoint that is deployed.
 * Extend the Ride core to easily fire REST calls against your service API and build intensive real-world test cases.
 
-There are additional test options avaialble as well:
```
Comment (tedcasey) [inline README.md:32]: Good catch.

File: README.md [hunk 5]
```diff
@@ -53,74 +53,74 @@ Architect/Driver: [Ted Casey](https://www.linkedin.com/in/ted-casey-0074064/)
 
 
 ## Repositories
-When extending Adobe Ride, these settings need to be added to the repository node of your project(s) (or parent project) in order to post your extension to your internal artifact management system.
+When extending Adobe Ride, these settings need to be added to the repository node of your project(s) (or parent project) to post your extension to your internal artifact management system.
```
Comment (tedcasey) [inline README.md]: This one I would leave as is.  I think the extra wordage helps.

File: README.md [hunk 6]
```diff
@@ -53,74 +53,74 @@ Architect/Driver: [Ted Casey](https://www.linkedin.com/in/ted-casey-0074064/)
...
+            </releases>
+            <snapshots>
+                <enabled>true</enabled>
+                <updatePolicy>never</updatePolicy>
+                <checksumPolicy>fail</checksumPolicy>
+            </snapshots>
+            <id>my-snapshots</id>
+            <name>my-snapshots</name>
+            <url>https://your.snapshot.artifact.repository:443/some/path//</url>
+        </repository>
```
Comment (tedcasey) [inline README.md:87]: Yes, Looks good.

File: README.md [hunk 7]
```diff
@@ -53,74 +53,74 @@ Architect/Driver: [Ted Casey](https://www.linkedin.com/in/ted-casey-0074064/)
...
 ``` 
 
 ## Settings
-Depending on your infrastructure you may need to add settings.xml file to your local .m2 directory in order for you to be able to download maven dependencies and build your extension.
+Depending on your infrastructure you may need to add settings.xml file to your local .m2 directory for you to be able to download maven dependencies and build your extension.
```
Comment (tedcasey) [inline README.md:91]: Ok.

File: README.md [hunk 8]
```diff
@@ -53,74 +53,74 @@ Architect/Driver: [Ted Casey](https://www.linkedin.com/in/ted-casey-0074064/)
...
+    </td></tr>
+    <tr><td>Fuzzer Library</td><td>Works in conjunction with the Model Utility to throw arrays of data (including edge and malicious data) at the api and determine what the right response should be</td><td>
+        <pre>&lt;dependency&gt;</br>&nbsp;&nbsp;&nbsp;&lt;groupId&gt;com.adobe.ride&lt;/groupId&gt;</br>&nbsp;&nbsp;&nbsp;&lt;artifactId&gt;ride-fuzzer-lib&lt;/artifactId&gt; </br>&nbsp;&nbsp;&nbsp;&lt;version&gt;0.0.1-SNAPSHOT&lt;/version&gt;</br>&lt;/dependency&gt;</pre>
+    </td></tr>
+    <tr><td>Performance Library</td><td>Scala libary based on Gatling which allows you to place your scala-modified test code in snippet block and run performance metrics</td><td>
+        <pre>&lt;dependency&gt;</br>&nbsp;&nbsp;&nbsp;&lt;groupId&gt;com.adobe.ride&lt;/groupId&gt;</br>&nbsp;&nbsp;&nbsp;&lt;artifactId&gt;ride-performance-lib&lt;/artifactId&gt; </br>&nbsp;&nbsp;&nbsp;&lt;version&gt;0.0.1-SNAPSHOT&lt;/version&gt;</br>&lt;/dependency&gt;</pre>
+    </td></tr>
```
Comment (tedcasey) [inline README.md:110]: Good.

File: README.md [hunk 9]
```diff
@@ -53,74 +53,74 @@ Architect/Driver: [Ted Casey](https://www.linkedin.com/in/ted-casey-0074064/)
...
+One of Adobe Ride's stated goals is the improved speed of rest API test development.  By measuring the number of logical lines of code required to create a given test with and without Ride, we can demonstratively show improvement in this regard.  We achieve this through the abstraction of repetitive code.  In essence, the more basic the test, the fewer lines of code should be required to write it in the framework.  This concept can be then further leveraged through extensibility.  By extending the architecture implemented in the core of the framework, even complex but regularly used logical paths can be compressed to fewer lines of code to achieve the same goal through extension.  Finally, we create the flexibility of usage through modularity.  The architecture is based (like many java projects) on hierarchically in/dependent jars, giving framework extenders options on what to extend.
```
Comment (tedcasey) [inline README.md:116]: Kind of stylistic, but ok.

File: README.md [hunk 10]
```diff
@@ -53,74 +53,74 @@ Architect/Driver: [Ted Casey](https://www.linkedin.com/in/ted-casey-0074064/)
...
 
 ### Abstraction and Optimization
 
 As discussed above one of the main goals of the framework is to reduce the amount of code required to write tests.  Below is an example of how this is achieved:
 
 
-POST Default Service Object in prep for test in Rest-Assured example (partial)
+POST Default Service Object in prep for a test in Rest-Assured example (partial)
```
Comment (tedcasey) [inline README.md]: This is a title for the code snippet, I think it is ok as is.

File: README.md [hunk 11]
```diff
@@ -137,7 +137,7 @@ statusCode(201).
 when().
   post("/binaryBody");
   
-//There is actually quite a bit more to define in terms of endpoints, completely define body, specify auth, etc.
```
Comment (tedcasey) [inline README.md:140]: This is kind of stylistic, but I suppose this can be removed.

File: README.md [hunk 12]
```diff
@@ -151,13 +151,13 @@ ServiceObject obj = new ServiceObject(false);
 // post object to server
 CoreController.post(obj);
 
-//All of the default config, endpoint specifying, etc. is handled in the core.  JSON data body is generated dynamically (you can change it in test as desired).
+//All of the default config, endpoint specifying, etc. is handled in the core.  JSON data body is generated dynamically (you can change it in the test as desired).
```
Comment (tedcasey) [inline README.md:154]: Ok

File: README.md [hunk 13]
```diff
@@ -151,13 +151,13 @@ ServiceObject obj = new ServiceObject(false);
...
 
 ```
 
 
 
-The drastically reduced amount of code required for the test setup, allows the test author to focus on what's important to his tests to the end user experience.
+The drastically reduced amount of code required for the test setup, allows the test author to focus on what's important to his tests to the end-user experience.
```
Comment (tedcasey) [inline README.md:160]: I think either is ok, but I guess we can change this.

File: README.md [hunk 14]
```diff
@@ -166,7 +166,7 @@ The diagram below lays out the structure of the architecture and how services ca
 ![Ride Flow](images/RideAutomationArchitecture.jpg)
 
  
-The diagram displays the components of the framework housed in the global corporate namespace, how they are build on top of each other, extended by a library within a business unit namespace designed to test a SaaS application within that namespace.
+The diagram displays the components of the framework housed in the global corporate namespace, how they are built on top of each other, extended by a library within a business unit namespace designed to test a SaaS application within that namespace.
```
Comment (tedcasey) [inline README.md:169]: good catch.

File: README.md [hunk 15]
```diff
@@ -175,20 +175,20 @@ Components:
 * config-util: setup endpoint targeting and test user authorization based on commands given at launch time.
 * core: rolls up all of the rest-assured groovy syntax into simple RestApiController.\<method> calls (i.e. RestApiController.put(req, resp) and handles the firing of the call based on the config-util input.
 * fuzzer-lib: exhaustively fuzzes a REST body with negative, edge-case, and malicious (though safe for testing) calls to the service.
-* performance-lib: Scala libary based on Gatling which allows you to place your scala-modified test code in snippet block and run performance metrics
+* performance-lib: Scala library based on Gatling which allows you to place your scala-modified test code in snippet block and run performance metrics
```
Comment (tedcasey) [inline README.md:178]: good catch.

File: README.md [hunk 16]
```diff
@@ -175,20 +175,20 @@ Components:
...
 
 ### Building and Release Versions
 Release versions are created as PRs are reviewed and made widely available. 
 
 
 ### Core Extensions -
-When a service REST API comes online, in order to test it with Adobe Ride, the Core will need to be extended to a service-library.  This is done in order to handle workflows and data specific to that API/Service.
+When a service REST API comes online, to test it with Adobe Ride, the Core will need to be extended to a service-library.  This is done to handle workflows and data specific to that API/Service.
```
Comment (tedcasey) [inline README.md:185]: Yes, this sounds better.

File: README.md [hunk 17]
```diff
@@ -175,20 +175,20 @@ Components:
...
+Shared resource is a term used to describe a set of resources you create for your Ride extension project (just like any other resource in a javaproject), but specifically json schema for data modeling and generation, and config files for use in endpoint targeting and testuser management. Ideally, your shared-resources resources lives in separate and centralized source repo outside of your extension project, built as a jar, and then used as a dependency for your project (explained in the usage docs). This is so that if, as is common, you are working in an environment creating multiple webservices/API, there can be one set of resources that can be used by multiple extensions for an easy end to end testing.  However, it is just as valid to put these resources in the resources directory of your java project (src/main/resources).
```
Comment (tedcasey) [inline README.md]: good catch.  Would you mind also changing "...lives in separate..." to "...lives in **a** separate..."

File: README.md [hunk 18]
```diff
@@ -175,20 +175,20 @@ Components:
...
 
 ### Reporting 
-Ride relies on TestNG and log4j for the bulk of reporting.  However use of ReportNG is a work in progress.
+Ride relies on TestNG and log4j for the bulk of reporting.  However, the use of ReportNG is a work in progress.
```
Comment (tedcasey) [inline README.md:191]: good catch.

File: README.md [hunk 19]
```diff
@@ -197,13 +197,13 @@ Usage docs can be found here: [Usage](Usage.md)
 Because TestNG is built into the framework, the functionality available there is applicable for annotating tests.  More information is available here: http://testng.org/doc/
 
 ### Command Line
-The built-in functionality of Maven/failsafe in addition to the aspects built into Ride, allows for simple execution of run tests as outlined below (from within the test repo root):
+The built-in functionality of Maven/failsafe in addition to the aspects built into Ride allows for simple execution of run tests as outlined below (from within the test repo root):
```
Comment (tedcasey) [inline README.md]: For this one, I think it reads better if you don't remove the comma, but add another comma after "failsafe"

File: README.md [hunk 20]
```diff
@@ -197,13 +197,13 @@ Usage docs can be found here: [Usage](Usage.md)
...
 
 ``
 mvn clean verify -Dtarget=<configname> -Psmoke
 ``
 
-In this case, the value supplied to the -Dtarget flag refers to the "xxx.config" file in your shared resources.  It contains all of the information for endpoint targeting.  The -Psmoke tag points to whatever information you put in your smoke profile in you maven test project pom.  All of this is explained further in the usage docs.
+In this case, the value supplied to the -Dtarget flag refers to the "xxx.config" file in your shared resources.  It contains all of the information for endpoint targeting.  The -Psmoke tag points to whatever information you put in your smoke profile in your maven test project pom.  All of this is explained further in the usage docs.
```
Comment (tedcasey) [inline README.md:206]: good catch.

**Review by tedcasey — CHANGES_REQUESTED** [review]
Thanks for reading through this!  A couple of minor changes requested.

Thanks again!  You're my new copywriter :)

File: README.md (hunk 19 above)
Comment (NinjaCoderDev) [reply README.md]: Did it in the next commit as you told.

File: README.md (hunk 3 above)
Comment (NinjaCoderDev) [reply README.md:25]: So i removed the 'a' as it is inappropriate here .

File: README.md (hunk 17 above)
Comment (NinjaCoderDev) [reply README.md]: Okk did it.

**Comment by NinjaCoderDev** [comment]
Please review this and comment if any change is required. Thanks :)

**Review by tedcasey — APPROVED** [review]
Looks good.  Thanks again!

Let me know if you want a T-shirt.  I'll send you one.  You can message me with your addr from the link in the readme.
